src/frontends/pytorch/src/op/var_mean.cpp

```diff
@@ -12,6 +12,8 @@
 #include "openvino/op/shape_of.hpp"
 #include "openvino/op/sqrt.hpp"
 #include "openvino/op/subtract.hpp"
+#include "openvino/op/unsqueeze.hpp"
```

Comment:
I do not see unsqueeze usage in this file? Does these headers added accidentally?